python/flatbuffers/number_types.py

```diff
@@ -170,3 +172,11 @@ def uint64_to_float64(n):
     packed = struct.pack("<1Q", n)
     (unpacked,) = struct.unpack("<1d", packed)
     return unpacked
+
+
+def to_numpy_type(number_type):
+    if np is not None:
+        return np.dtype(number_type.name).newbyteorder('<')
+    else:
+        raise RuntimeError(('Numpy could not be imported'
```

Comment:
I updated to have a custom exception NumpyRequiredForThisFeature that is defined in compat.py, but I decided I didn't actually like putting a function there that throws the error. It's only repeated twice, and would make debugging slightly harder (one extra layer of indirection in the stack trace) if the error was raised.